### ISSUE TITLE
add option to disable trim current line on autosave

### ISSUE THREAD

**Issue by aboettger-tuhh** [shardulm94/vscode-trailingspaces#42]
Please add an option 
- [ ] dont't  trim current line on autosave

**Comment by seanhamlin** [comment]
Yes please. This is super annoying when autosave is on (even with a delay). I guess a 'fix' is to not autosave as often, however I would much prefer the trimmer to have the above mentioned option. That would be ace.

**Comment by propertone** [comment]
Thoughts on fixing this? Would you be open to a PR contributing this change?
